### PR TITLE
Updated two paragraphs for readability

### PR DESCRIPTION
**Pull request by Fabijenna**

### CODE REVIEW

File: webpage/src/contributing/translation.md [hunk 1]
```diff
@@ -109,6 +107,6 @@ Some segments we don't translate in the Documentation:
 
 You will deal with the language of this software, learn new terms and receive a lot of inside information about the project.
 
-If you are thus ready to help to proofread the translation of your language, [Amy Fabijenna](https://crowdin.com/profile/rawfreeamy) welcomes you to a short briefing in the practice of translating and proofreading, where you can ask her anything to help you to have fun contributing. If you have no or only little experience translating, you can book a completely cost free Zoom introduction into translation. 
+If you are thus ready to help to proofread the translation of your language, [Amy Fabijenna](https://crowdin.com/profile/rawfreeamy) welcomes you to a short briefing in the practice of translating and proofreading. You can ask her anything that helps you having fun contributing.
```
Comment (pbek) [inline webpage/src/contributing/translation.md:110]: My grammar check again complains about "having".

Comment (Fabijenna) [reply webpage/src/contributing/translation.md:110]: You have a great grammar checker. What is it?

**Review by Fabijenna — COMMENTED** [review]
please change "having" to "have" or let me know how I can do it

**Comment by Fabijenna** [comment]
of course, it needs to be "have" fun.

File: webpage/src/contributing/translation.md (hunk 1 above)
Comment (pbek) [reply webpage/src/contributing/translation.md:110]: Integrated in the Jetbrains suits...

**Pull request merged**